Декоративные элементы на главной странице

Ierixorn opened the pull request:
---
:mortar_board: [Декоративные элементы на главной странице](https://up.htmlacademy.ru/htmlcss/33/user/1898375/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/1898375-gllacy-33/9/

Ierixorn committed:
Добавление декоративных элементов

Добавление svg элементов и работа с кнопками на главной

Ierixorn committed:
Добавление декоративных элементов (продолжение)

Доработка кнопок, добавление декоративных элементы для слайдера

Ierixorn committed:
Добавлены декоративные элементы и доработана стилизация

Добавление декоративных элементов и исправление / добавление стилизации главной странице

Ierixorn committed:
Добавление декоративных элементов странице каталога

Работа над "крошками"  и фильтрами.

Ierixorn committed:
Добавление декоративных элементов странице каталога (продолжение)

sealeye commented on css/style.css:
```diff
@@ -141,34 +205,92 @@ body {
 	line-height: 20px;
-	color: #2D3440;
 	text-decoration: none;
+  background-color: #FFCBD8;
```
В макете фон серый с прозрачностью и при наведении/нажатии меняется прозрачность фона

sealeye commented on css/style.css:
```diff
@@ -141,34 +205,92 @@ body {
-  background-color: #FFCBD8;
-  border-radius: 50%;
-  margin: 0;
+  position: absolute;
```
Вместо позиционирования нужно использовать Флекс для родителя. 
Это касается всех иконок

sealeye commented on css/style.css:
```diff
@@ -99,36 +133,66 @@ body {
 
+.navigation-button:hover {
+  border-radius: 26px;
+  background-color: #FFCBD8;
```
Тут должен быть светлый фон с прозрачностью, а не розовый

sealeye commented on css/style.css:
```diff
@@ -391,11 +667,40 @@ body {
+
+.basket-thumb:hover {
+  background-color: #FCFCFC;
+  outline: 2px solid #FF2F64;
```
тут должен быть бордер, потому что в макете линия внутри ширины, а не снаружи

sealeye replied:
при фокусе тоже

sealeye reviewed (commented):
в футере все ссылки должны иметь эффекты наведения и нажатия 
все ссылки/кнопки должны иметь состояние active

Ierixorn committed:
Добавление декоративных элементов каталога и исправлений

Декоративные элементы каталога (кроме range фильтра)
Доработка элементов главной.
